hazelcast/src/test/java/com/hazelcast/test/TestHazelcastInstanceFactory.java

```diff
@@ -187,7 +186,7 @@ protected static Address createAddress(String host, int port) {
 
     private static Config init(Config config) {
         if (config == null) {
-            config = new XmlConfigBuilder().build();
+            config = new Config(); //new XmlConfigBuilder().build();
```

Comment:
Do we need the commented part? If not; it is better to get rid of it. And afaik they both will load the same config.